File: SampleCode-V5/android-sdk-v5-uxsdk/src/main/java/dji/v5/ux/payload/data/NetworkConnectionState.java
package dji.v5.ux.payload.data;

/**
 * 网络连接状态枚举
 */
public enum NetworkConnectionState {
    DISCONNECTED,    // 未连接
    CONNECTING,      // 连接中
    CONNECTED,       // 已连接
    ERROR            // 错误状态
}

File: SampleCode-V5/android-sdk-v5-sample/src/main/java/dji/sampleV5/aircraft/data/NetworkConnectionState.kt
package dji.sampleV5.aircraft.data

/**
 * 网络连接状态枚举
 * 
 * @author Payload开发
 * @date 2024/1/1
 * 
 * Copyright (c) 2024, DJI All Rights Reserved.
 */
enum class NetworkConnectionState {
    DISCONNECTED,    // 未连接
    CONNECTING,      // 连接中
    CONNECTED,       // 已连接
    ERROR            // 错误状态
}